docs/source/apidocs/wavefunction.rst

```diff
@@ -1,8 +1,11 @@
+:orphan:
```

Comment:
This change was motivated by the following warning:

```
checking consistency... docs/source/apidocs/wavefunction.rst: WARNING: document isn't included in any toctree
```

Another option would be to instead include this document in the "API Reference"  `toctree` in `source/index.rst`.